aplt/scenarios.py

```diff
@@ -158,19 +160,75 @@ def notification_forever_stored(qty_stored=1000, ttl=300, flood_delay=30,
         for i in range(qty_stored):
             response, content = yield send_notification(endpoint, data, ttl)
             yield counter("notification.throughput.bytes", length)
-            yield counter("notification.sent", i)
-            notif = yield expect_notification(reg["channelID"], ttl)
-            yield counter("notification.received", 1)
-            message_ids.append(notif["version"])
-            yield wait(notif_delay)
+            yield counter("notification.sent", 1)
 
-        yield wait(flood_delay)
+        yield wait(5)
```

Comment:
Not the `flood_delay` ?

Reply:
Right, the flood_delay is waited on later.

Reply:
Ah, must be outside of this function, since I don't see any references to flood_delay here. Could probably drop it as an arg eventually.